### PR TITLE
NEWS: file update

### PR DESCRIPTION
**Pull request by shamisp**
Signed-off-by: Pavel Shamis (Pasha) <pasharesearch@gmail.com>

### CODE REVIEW

**Comment by shamisp** [comment]
@yosefe @bbenton @miked-mellanox please help me with v1.2 news file for the release

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1857/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3825/ for details (Mellanox internal link).

File: NEWS [hunk 1]
```diff
@@ -6,6 +6,21 @@
 ##
 #
 
+## 1.2.0 (June 13, 2017)
+
+Supported platforms
+  - Shared memory: KNEM, CMA, XPMEM, SYSV, Posix
```
Comment (yosefe) [inline NEWS]: no need - it's already listed in 1.1 / 1.0

File: NEWS [hunk 2]
```diff
@@ -6,6 +6,21 @@
...
+  - RDMA interconnects with VERBs interface (InfiniBand, RoCE, iWarp, Omni-Path),
+    Cray Gemini, Cray Aries, Mellanox ConnectX4/5
```
Comment (yosefe) [inline NEWS]: No need to mention explicitly which Mellanox devices are supported, it falls under "InfiniBand" category

File: NEWS [hunk 3]
```diff
@@ -6,6 +6,21 @@
...
+  - ISAs: x86, x86_64, ARM 64bit, Power
```
Comment (yosefe) [inline NEWS]: ISAs->Architectures.
Also, x86 is not supported, only x86_64
Power->Power64

File: NEWS [hunk 4]
```diff
@@ -6,6 +6,21 @@
...
+Features:
+  - added support for InfiniBand DC and UD transport including native MLX5 architecture support
```
Comment (yosefe) [inline NEWS]: - Added support for InfiniBand DC and UD transports, including accelerated verbs for Mellanox devices
- Full support for PGAS/SHMEM interfaces, blocking and non-blocking APIs
- Support for MPI tag matching, both in software and offload mode
- Zero copy protocols and rendezvous, registration cache
- Handling transport errors
- Flow control for DC/RC
- Dataypes support: contiguous, IOV, generic
- Multi-threading support

File: NEWS [hunk 5]
```diff
@@ -6,6 +6,21 @@
...
+  - support for ARM 64bit platforms
```
Comment (yosefe) [inline NEWS]: pls use Capital letter for consistency

**Comment by yosefe** [comment]
bot:ornl:retest

File: NEWS (hunk 2 above)
Comment (shamisp) [inline NEWS]: okay

File: NEWS (hunk 3 above)
Comment (shamisp) [inline NEWS]: good point

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1859/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3827/ for details (Mellanox internal link).

File: NEWS [hunk 6]
```diff
@@ -1,19 +1,56 @@
...
 ##
 ## See file LICENSE for terms.
 ##
 #
 
+## 1.2.0 (June 13, 2017)
+
+Supported platforms
+  - Shared memory: KNEM, CMA, XPMEM, SYSV, Posix
+  - RDMA interconnects with VERBs interface (InfiniBand, RoCE, iWarp, Omni-Path),
```
Comment (yosefe) [inline NEWS]: shall we remove iWarp, Omni-Path since it was not tested?

Comment (shamisp) [inline NEWS]: agreed

**Comment by shamisp** [comment]
@bbenton please take a look

**Comment by bbenton** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1870/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1871/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3838/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3839/ for details (Mellanox internal link).

**Pull request merged**